src/Java.Interop/Java.Interop/JavaObject.cs

```diff
@@ -143,10 +134,6 @@ public override unsafe string ToString ()
 			return JniEnvironment.Strings.ToString (ref lref, JniObjectReferenceOptions.CopyAndDispose);
 		}
 
-		JniManagedPeerStates IJavaPeerable.JniManagedPeerState {
```

Comment:
@jonpryor Not sure whether having the same explicit interface getter and public getter methods of the same name was an intention here? Hopefully not.